tests/test_core.py

```diff
@@ -900,40 +900,42 @@ def test_setjmp_noleak(self):
     self.do_run(src, r'''ok.''')
 
   def test_exceptions(self):
-    self.set_setting('EXCEPTION_DEBUG', 1)
-    # needs to flush stdio streams
-    self.set_setting('EXIT_RUNTIME', 1)
+    for support_longjmp in [0, 1]:
+      self.set_setting('SUPPORT_LONGJMP', support_longjmp)
+      self.set_setting('EXCEPTION_DEBUG', 1)
```

Comment:
the options that are the same in each iteration of the loop may as well be outside of the loop, for clarity